developer_manual/client_apis/OCS/ocs-api-overview.rst

```diff
@@ -86,11 +136,12 @@ Clients can obtain capabilities provided by the Nextcloud server and its apps vi
      </meta>
      <data>
       <version>
-       <major>12</major>
+       <major>17</major>
```

Comment:
You used tabs in the example above (lines ~100), shall we use them here too?

Reply:
Sure, that makes sense. Thank you for the fast review!
I have changed all indentations in the document to tabs.